readtags-cmd/readtags-cmd.c

```diff
@@ -136,40 +136,59 @@ static void listTags (int pseudoTags)
 static const char *const Usage =
 	"Find tag file entries matching specified names.\n\n"
 	"Usage: \n"
-	"    %s -h\n"
-	"    %s [-ilp] [-n] "
+	"    %s -h | --help\n"
+	"        Print this help message.\n"
 #ifdef READTAGS_DSL
-	"[-Q EXP] "
+	"    %s -HQ| --help-expression filter\n"
+	"        Print available operators in filter expression.\n"
 #endif
-	"[-s[0|1]] [-t file] [-] [name(s)]\n\n"
+	"    %s [OPTIONS] ACTION\n"
+	"        Do the specified action.\n"
+	"Actions:\n"
+	"    -l | --list\n"
+	"        List regular tags.\n"
+	"    [-] name...\n"
+	"        List regular tags matching the name.\n"
+	"        \"-\" indicates arguments after this as NAME even if they start with -.\n"
```

Comment:
I'm not sure about why NAME is in uppercase. If it doesn't have a special meaning, this may be better:

```suggestion
	"        \"-\" indicates arguments after this as names even if they start with -.\n"
```

Reply:
I should write like:
```
	"    [-] NAME\n"
	"        List regular tags matching NAME.\n"
	"        \"-\" indicates arguments after this as NAME even if they start with -.\n"
```